src/optigraph.jl

```diff
@@ -936,6 +956,26 @@ function JuMP.termination_status(graph::OptiGraph)
     return MOI.get(graph.moi_backend, MOI.TerminationStatus())
 end
 
+function JuMP.primal_status(graph::OptiGraph)
+    return MOI.get(graph.moi_backend, MOI.PrimalStatus())
+end
+
+function JuMP.dual_status(graph::OptiGraph)
+    return MOI.get(graph.moi_backend, MOI.DualStatus())
+end
+
+function JuMP.callback_value(cb_data, x::GenericVariableRef, graph::OptiGraph)
```

Comment:
The retrieved callback values are not updated by the optimizer. The code below works:
```
function JuMP.callback_value(cb_data, x::GenericVariableRef, graph::OptiGraph)
    var_index = graph.moi_backend.model_to_optimizer_map.var_map[x]
    return MOI.get(
        graph.moi_backend.optimizer,
        MOI.CallbackVariablePrimal(cb_data),
        var_index,
    )
end
```

Reply:
thanks @tso-martin. I will make a new PR to fix this (with tests to boot).